.eslintrc

```diff
@@ -1,46 +1,48 @@
 {
   "extends": ["standard"],
-    "env": {
```

Comment:
This "env" was indented wrong, most of this was is a result of that indentation.